Exporter/src/main/java/uk/ac/ebi/biostd/exporter/jobs/partial/PartialSubmissionExporter.java

```diff
@@ -93,13 +94,21 @@ private String writeFile(List<Submission> submissions) {
         String fullFilePath = getFileName();
         Files.deleteIfExists(Paths.get(fullFilePath));
 
+        long startTimeTS = System.currentTimeMillis();
         try (BufferedWriter bw = new BufferedWriter(new FileWriter(fullFilePath))) {
+            // Since we are delegating the actual writing to jackson, the end time is always going to be equal to the
+            // start time here. TODO: Refactor Partial/Full exporters to use the same code
```

Comment:
@awaisathar sorry for the delay this ticket so much. I think as this is not critical we should implement TODO as part of ticket to avoid tech debt. Let me know if you find complicated due to you other assignments,   I can help with design or completing changes.